packages/app-bridge/src/react/useBlockTemplates.ts

```diff
@@ -10,6 +10,7 @@ export const useBlockTemplates = (appBridge: AppBridgeBlock) => {
     const blockId = appBridge.getBlockId();
 
     const [blockTemplates, setBlockTemplates] = useState<Record<string, Template[]>>({});
+    const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
Suggestion:
- Rename to just `error`
- Use `null` when there are no error

```suggestion
    const [error, setError] = useState<string | null>(null);
```

Reply:
Got it. Refactored.
I also pushed non-error related thing. 
Just found that I've missed to add the 'openNewPublication' command name to the `BlockCommand` type in `AppBridgeBlock.ts`. 
Though it will be an overkill to do the separate PR for a one liner fix.